beacon-chain/db/filesystem/data_column.go

```diff
@@ -0,0 +1,966 @@
+package filesystem
+
+import (
+	"context"
+	"encoding/binary"
+	"encoding/hex"
+	"fmt"
+	"io"
+	"os"
+	"path"
+	"path/filepath"
+	"strconv"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/pkg/errors"
+	"github.com/prysmaticlabs/prysm/v5/async"
+	"github.com/prysmaticlabs/prysm/v5/async/event"
+	fieldparams "github.com/prysmaticlabs/prysm/v5/config/fieldparams"
+	"github.com/prysmaticlabs/prysm/v5/config/params"
+	"github.com/prysmaticlabs/prysm/v5/consensus-types/blocks"
+	"github.com/prysmaticlabs/prysm/v5/consensus-types/primitives"
+	"github.com/prysmaticlabs/prysm/v5/io/file"
+	ethpb "github.com/prysmaticlabs/prysm/v5/proto/prysm/v1alpha1"
+	"github.com/prysmaticlabs/prysm/v5/time/slots"
+	"github.com/spf13/afero"
+)
+
+const (
+	version                                    int = 0x01
+	versionSize                                    = 1           //bytes
+	maxSszEncodedDataColumnSidecarSize             = 536_870_912 // 2**(4*8) / 8 bytes
+	encodedSszEncodedDataColumnSidecarSizeSize     = 4           // bytes (size of the endoded size of the SSZ encoded data column sidecar)
```

Comment:
`endoded`

Reply:
Fixed in https://github.com/prysmaticlabs/prysm/pull/15061/commits/5dfef04a6b81ecadc7d2ded43241745a97de4aaa.